<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewTab</class>
 <widget class="QWidget" name="ViewTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frameButtons">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>243</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Choose what to display:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showTabbedButton">
        <property name="text">
         <string>Tabbed multi-view widget</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="singleViewButton">
        <property name="text">
         <string>Single render view</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="spreadsheetViewButton">
        <property name="text">
         <string>Spreadsheet view</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameView">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutView">
      <item>
       <widget class="QWidget" name="widget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>singleViewButton</sender>
   <signal>clicked()</signal>
   <receiver>ViewTab</receiver>
   <slot>insertSingleView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showTabbedButton</sender>
   <signal>clicked()</signal>
   <receiver>ViewTab</receiver>
   <slot>insertMultiView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spreadsheetViewButton</sender>
   <signal>clicked()</signal>
   <receiver>ViewTab</receiver>
   <slot>insertSpreadsheetView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>insertMultiView()</slot>
  <slot>insertSingleView()</slot>
  <slot>insertSpreadsheetView()</slot>
 </slots>
</ui>
